### ISSUE TITLE
I'm not less sure what's going on when there's nothing in the details element to expand. Its just an empty tab stop at that point. Those aren't great, but they're not an WCAG violation IMO.

### ISSUE THREAD

**Issue by tony1111p** [dequelabs/axe-core#4569]
I'm not less sure what's going on when there's nothing in the details element to expand. Its just an empty tab stop at that point. Those aren't great, but they're not an WCAG violation IMO.

_Originally posted by @WilcoFiers in https://github.com/dequelabs/axe-core/pull/4511#discussion_r1652373615_

**Comment by straker** [comment]
Thanks for the ticket. Was there something you wanted to address in addition to Wilco's comment? Otherwise I'll close the ticket since it's neither a question nor an issue.
